dg-common/src/main/java/org/finra/datagenerator/common/SocialNetwork_Example_Java/UserTransitions.java

```diff
@@ -1,3 +1,4 @@
+//CHECKSTYLE:OFF
```

Comment:
Please remove. If checkstyle rules are broken in the java source, then please fix the java source.